Assets/SteamVR_Unity_Toolkit/Scripts/VRTK_SimplePointer.cs

```diff
@@ -22,6 +22,8 @@ public class VRTK_SimplePointer : VRTK_WorldPointer
         public float pointerLength = 100f;
         public bool showPointerTip = true;
         public GameObject customPointerCursor;
+        [Tooltip("Check this if you would like to keep the custom pointer cursor material.")]
+        public bool keepCustomPointerMaterial = false;
```

Comment:
customPointerCursor is implemented by them, instead of WorldPointer, to they probably should stick together.
I think it would be better to leave there for now and move customPointerCursor and keepCustomPointerMaterial to WorldPointer in another commit.